ui/DesignSystem/Component/BigHeader.svelte

```diff
@@ -68,7 +68,7 @@
     height: var(--topbar-height);
     width: 100%;
     max-width: var(--content-max-width);
-    padding: 0 var(--content-padding);
+    padding: 0 var(--content-padding) 0 var(--content-padding);
```

Comment:
`padding: 0 var(--content-padding);` is the same as `padding: 0 var(--content-padding) 0 var(--content-padding);`

Reply:
@juliendonck it is the same but with less code. is there a specific reason why you would prefer the extended version?

Reply:
Oh maybe my comment wasn't clear. Since the change is from the short to the extended version, I wanted to point out that that's unnecessary

Reply:
Ah, sorry! I thought this was a code suggestion to extend it 🤦‍♂️ is goed! Sorry for the noise